.travis.yml

```diff
@@ -1,41 +1,24 @@
-#language: shell
+language: bash
 
-# Required environment variables from repository settings
```

Comment:
I've opened a support ticket securiteinfo.com.
I hope they will give a Pro license for running Travis.